mac/sound.cpp

```diff
@@ -103,24 +103,24 @@ void CSound::GetAvailableInOutDevices()
 
     // always add system default devices for input and output as first entry
     lNumDevs                 = 0;
-    strDriverNames[lNumDevs] = "System Default In/Out Devices";
+    strDriverNames[lNumDevs] = "System Default In/Out Devices"; // TODO: why can't this be translated?
```

Comment:
According to the language style guide shouldn't this be sound hardware?

```suggestion
    strDriverNames[lNumDevs] = "System default in/out hardware"; // TODO: why can't this be translated?
```

Reply:
Ah yes - maybe :-) I added some definitions to the style guide in the hope that they would be possible in the actual text we have. So if the style guide rules make sense in this case, we should apply them.

Reply:
What is the string used for?  It appears to be being passed out of Jamulus to some other system.  How is that system using the string?  Is the value of the string important?

Reply:
Line 275 says `if ( strDriverName.compare ( strDriverNames[i] ) == 0 )`, so leave `strDriverNames[lNumDevs]` alone.